### PR TITLE
[ci] No need to start memcached for tests

### PR DESCRIPTION
**Pull request by coolo**
The rails cache store used is mem_store, which is purely
in process. Dalli is only used in product and development

### CODE REVIEW

**Comment by coolo** [comment]
required since 2013 (d6ba1c5b2c24e3ba3294d55ed1716c98e0dd07e7) obsolete since 2016 b8d0eddb68c7049eca2290eb16597ad5d4b511ea - but we even have a useless container now :)

And we have minitests that fail without memcached - which is the most suprising part

**Comment by coolo** [comment]
(the container is only unused for CI, but required/useful for dev)

File: src/api/lib/tasks/dev.rake [hunk 1]
```diff
@@ -20,7 +20,6 @@ namespace :dev do
     copy_example_file('config/options.yml')
     options_yml = YAML.load_file('config/options.yml') || {}
     options_yml['source_host'] = args.old_test_suite ? 'localhost' : 'backend'
-    options_yml['memcached_host'] = 'cache'
```
Comment (ChrisBr) [inline src/api/lib/tasks/dev.rake]: This is still needed for development

Comment (coolo) [reply src/api/lib/tasks/dev.rake]: yeah, will fix that up

**Pull request merged**